app/javascript/controllers/auto_citation_controller.js

```diff
@@ -55,19 +55,23 @@ export default class extends Controller {
   // Authors (person and organization) as an array of strings.
   get authors() {
     return this.contributorRoles.map((roleField, index) => {
-      if (roleField.value.startsWith('person')) {
+      if (roleField.attributes['data-contributors-target'].value == 'selectPersonRole') {
         const firstInitial = `${this.contributorFirsts[index].value.charAt(0)}.`
```

Comment:
This is how we know we are dealing with a person and not an organization (since we don't have that prepended "person|" stuff anymore